packages/webapp/src/webview/ui/components/Header/NavigationButtons/NavigationButtons.tsx

```diff
@@ -7,7 +7,15 @@ import i18next from 'i18next';
  *
  * @returns A button component navigating to all answers.
  */
-export function AllAnswersButton() {
+const AllAnswersButton = React.memo(function AllAnswersButton() {
+    requestAnimationFrame(() => {
+        //(document.body.querySelector('button') as HTMLElement).focus();
```

Comment:
Maybe remove this?

Reply:
Thanks for pointing that out. The comment has now been removed